core/src/main/java/com/linecorp/armeria/server/annotation/ResponseConverterFunction.java

```diff
@@ -75,4 +76,13 @@ static <T> T fallthrough() {
         // Always throw the exception quietly.
         throw FallthroughException.get();
     }
+
+    /**
+     * Create a ResponseConverterFunction which, during response conversion, will use the current instance
+     * to convert the response. If that fails, the otherConverter will be used
+     * @param otherConverter the ResponseConverterFunction to be used in case the current instance is unable to convert the response
+     */
+    default OrElseResponseConverterFunction orElse(ResponseConverterFunction otherConverter) {
+        return new OrElseResponseConverterFunction(this, otherConverter);
```

Comment:
Let's add
`requireNonNull(otherConverter, "otherConverter");`

Reply:
This passes without any additional code https://github.com/line/armeria/pull/4312/files#r907068352

Looks like it's already configured to be `@NonNullByDefault`